proto/viam/app/mltraining/v1/ml_training.proto

```diff
@@ -21,7 +21,7 @@ enum ModelType {
 
 message SubmitTrainingJobRequest {
   viam.app.data.v1.Filter filter = 1;
-  string organization_id = 2;
+  string org_id = 2;
```

Comment:
Should actually be organization_id as we're moving to organization_id instead of org_id for consistency across packages